### PR TITLE
[RW-9518][risk=no]   [tests] Cromwell / User App tests - Using R Notebook

### PR DESCRIPTION
**Pull request by NehaBroad**
Apart from Python, this PR will use R notebook to check
-   cromshell is running
- The status of cromshell (provisioning or running)
- Upload wdl and json file
- Submit wdl using cromshell
- Clean up: delete cromshell, notebook and workspace

**Description**

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
- [ ] If this PR is intended to complete a JIRA story, I have checked that all AC are met for that story
- [x] I have manually tested this change and my testing process is described above
- [x] This PR includes appropriate automated tests, and I have documented any behavior that cannot be tested with code
- [ ] If this fixes a bug, ensure the steps to reproduce are in the Jira ticket or provided above.
- [x] I have added explanatory comments where the logic is not obvious
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented the impacts in the description
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this includes an API change, I have run the relevant E2E tests locally because API changes are not covered by our PR checks

### CODE REVIEW

File: e2e/tests/nightly/gke-apps/cromwell.spec.ts [hunk 1]
```diff
@@ -16,6 +16,10 @@ const fileBasePath = '../../../../resources/cromwell/';
 const wdlFilePath = path.relative(process.cwd(), __dirname + fileBasePath + wdlFileName);
 const jsonFilePath = path.relative(process.cwd(), __dirname + fileBasePath + jsonFileName);
 
+const cromshellSubmitPythonCmd = `!%s submit ${wdlFileName} ${jsonFileName}`;
+const cromshellSubmitRCmd =
+  `system2('%s', args = c('submit', '${wdlFileName}','${jsonFileName}'), ` + ' stdout = TRUE, stderr = TRUE)';
```
Comment (Peter-Lavigne) [inline e2e/tests/nightly/gke-apps/cromwell.spec.ts:23]: Minor: I find the combined use of string concatenation, backtick-syntax, and `%s` confusing here.

Maybe these could be functions?

```
const cromshellSubmitRCmd = cromshell_version =>
  `system2('${cromshell_version}', args = c('submit', '${wdlFileName}','${jsonFileName}'), stdout = TRUE, stderr = TRUE)`;
```

**Review by Peter-Lavigne — APPROVED** [review]
LGTM! Since this has passed for you locally I think this is OK.

Somewhat related, I just noticed our CI didn't run this code at all 😱. Our CI skips running E2E tests if the UI code hasn't changed, but doesn't consider the E2E tests. I created [a ticket](https://precisionmedicineinitiative.atlassian.net/browse/RW-10039) for that.

File: e2e/tests/nightly/gke-apps/cromwell.spec.ts [hunk 2]
```diff
@@ -60,7 +66,10 @@ describe('Cromwell GKE App', () => {
     await workspaceDataPage.deleteWorkspace();
   });
 
-  test('Run cromwell using python notebook', async () => {
+  test.each([
+    [Language.Python, 'All of Us Cromwell Setup Python snippets', cromshellSubmitPythonCmd],
+    [Language.R, 'All of Us Cromwell Setup snippets', cromshellSubmitRCmd]
+  ])('Run cromwell using notebook', async (language, snippetMenu, cromshellSubmitCommand) => {
```
Comment (Peter-Lavigne) [inline e2e/tests/nightly/gke-apps/cromwell.spec.ts:72]: The test name should include the programming language so we can tell which language fails the test when this runs.

**Review by jmthibault79 — APPROVED** [review]
Tests passed locally for me, after a whopping 45 min.  Phew!

**Pull request merged**